assets/src/blocks/HubspotForm/HubspotFormContext.js

```diff
@@ -0,0 +1,69 @@
+import {
+  createContext,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from '@wordpress/element';
+
+const Context = createContext({});
+Context.displayName = 'HubspotFormContext';
+const { Provider, Consumer } = Context;
+
+const ContextProvider = ({
+  children,
+}) => {
+  const wrapperRef = useRef(null);
+  const [ blockHeight, setBlockHeight ] = useState(0);
+
+  const updateBlockHeight = useCallback(() => {
+    if(wrapperRef && wrapperRef.current) {
+      console.log(wrapperRef.current.getBoundingClientRect().height)
+      setBlockHeight(wrapperRef.current.getBoundingClientRect().height);
+    }
+  }, [
+    blockHeight,
+    wrapperRef.current,
+  ]);
+
+  const onResizeHandler = useCallback((evt) => {
+    evt.preventDefault();
+    updateBlockHeight();
+  }, []);
+
+  useEffect(() => {
+    console.log(wrapperRef);
+    updateBlockHeight();
+  }, [
+    wrapperRef,
+  ])
```

Comment:
Why do we need an effect to run on every render to update the block height?

Reply:
Not on every render, this effect is invoked only when the wrapperRef change. In this case only once or twice but no more.

Reply:
Ah sorry I misread the dependencies.